models/user.js

```diff
@@ -0,0 +1,42 @@
+const bcrypt = require('bcrypt')
+const mongoose = require('mongoose')
+
+const UserSchema = new mongoose.Schema({
+    email: {
+        type: String,
+        required: true,
+        unique: true,
+        match: [/^.+@(?:.+\.)?ua\.edu$/, 'Email must be a valid ua.edu address']
+    },
+    password: {
+        type: String,
+        required: true,
+        minlength: 8
+    }
+})
```

Comment:
Should we create more fields? First name? Last name? any others? For signup frontend page I created option for first and last name to be entered